### PR TITLE
Dutch language integration

### PR DESCRIPTION
**Pull request by GHim9182**

### CODE REVIEW

**Comment by GHim9182** [comment]
Hi @MaciejLipinski, could you kindly take a look at this pull request? I'm available for discussions.

**Comment by jglaszka** [comment]
Hi @GHim9182, sorry for no answer - temporarily there was no person taking care of this project. 
I will take a look at your code this week.

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        2000       | "tweeduizend"
+        3000       | "drieduizend"
+        4000       | "vierduizend"
+        5000       | "vijfduizend"
+        7634       | "zevenduizendzeshonderdvierendertig"
+
+        11000      | "elfduizend"
+        15000      | "vijftienduizend"
+        21000      | "éénentwintigduizend"
+        24190      | "vierentwintigduizendhonderdnegentig"
```
Comment (jglaszka) [inline src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy:91]: Can you add test case also for 24191?

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        60         | "zestig"
+        70         | "zeventig"
+        80         | "tachtig"
+        90         | "negentig"
+
+        21         | "éénentwintig"
+        37         | "zevenendertig"
+        43         | "drieënveertig"
+        58         | "achtenvijftig"
+        69         | "negenenzestig"
```
Comment (jglaszka) [inline src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy:56]: Can you add test case also for 62?

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        272        | "tweehonderdtweeënzeventig"
+        387        | "driehonderdzevenentachtig"
+        448        | "vierhonderdachtenveertig"
+        569        | "vijfhonderdnegenenzestig"
+        625        | "zeshonderdvijfentwintig"
+        782        | "zevenhonderdtweeëntachtig"
+        895        | "achthonderdvijfennegentig"
+        999        | "negenhonderdnegenennegentig"
+
+        1000       | "duizend"
```
Comment (jglaszka) [inline src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy:81]: Please add test case also for 1001

**Review by jglaszka — CHANGES_REQUESTED** [review]
Before I will check whole code, we need to add more test cases in unit tests.

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy (hunk 1 above)
Comment (GHim9182) [reply src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy:93]: Done

File: src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy (hunk 3 above)
Comment (GHim9182) [reply src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/dutch/DutchValuesTest.groovy:82]: Done

**Comment by jglaszka** [comment]
I will take a look at this today.

**Review by jglaszka — APPROVED** [review]
Looks good to me. :) tommorow I will merge this

**Pull request merged**